wit/caching.wit

```diff
@@ -16,28 +16,28 @@ interface cache {
 
     // The `get` operation returns the value passed by a previous `set` for the
     // same key within the given TTL or none if there is no such value.
-    get: func(k: key) -> future-get-result;
+    get: func(k: key) -> borrow<future-get-result>;
```

Comment:
Would it make sense to return an owned `future-get-result` resource here for consuming?